tests/raft/test_raft.rs

```diff
@@ -240,24 +241,27 @@ pub fn new_message(from: u64, to: u64, t: MessageType, n: usize) -> Message {
     if n > 0 {
         let mut ents = Vec::with_capacity(n);
         for _ in 0..n {
-            ents.push(new_entry(0, 0, SOME_DATA));
+            ents.push(new_entry(0, 0, SOME_DATA, false));
         }
         m.set_entries(RepeatedField::from_vec(ents));
     }
     m
 }
 
 pub fn empty_entry(term: u64, index: u64) -> Entry {
-    new_entry(term, index, None)
+    new_entry(term, index, None, true)
 }
 
-pub fn new_entry(term: u64, index: u64, data: Option<&str>) -> Entry {
+pub fn new_entry(term: u64, index: u64, data: Option<&str>, empty_entry: bool) -> Entry {
     let mut e = Entry::new();
     e.set_index(index);
     e.set_term(term);
     if let Some(d) = data {
         e.set_data(d.as_bytes().to_vec());
     }
+    if !empty_entry {
```

Comment:
why using empty_entry to determine sync or not?

Reply:
all entries in test_raft are not sync,  but we may have entry.has_sync_log() == false

Reply:
I still can't get the point... 

The default sync_log is false for the event, why do you set to false again with not empty?